packages/app/package.json

```diff
@@ -80,10 +80,10 @@
       "tvosTarget": "13.0"
     },
     "android": {
-      "minSdk": 21,
+      "minSdk": 23,
       "targetSdk": 34,
       "compileSdk": 34,
-      "firebase": "33.16.0",
+      "firebase": "34.0.0",
```

Comment:
app distribution is at beta16 now and need a manual edit in build.gradle, these are unfortunately manual since they aren't in the BoM unfortunately (I just re-tested now without the `:16.0.0-beta15` version on there to see if it worked but nope)

[  implementation 'com.google.firebase:firebase-appdistribution-api:16.0.0-beta15'
](https://github.com/invertase/react-native-firebase/blob/fe4550f362548dcb66359044715e22222a413f97/packages/app-distribution/android/build.gradle#L99-L103)